### PR TITLE
Update zha.markdown to clearify zigpy-cc & zigpy-zigate as experimental

### PR DESCRIPTION
**Pull request by Gamester17**
## Proposed change

Update zha.markdown to clarify that zigpy-cc and zigpy-zigate are should still be considered as experimental.  This is as per zigpy README.md https://github.com/zigpy/zigpy/blob/dev/README.md

Some people might get the wrong impression that ZHA is unstable with those without statement like this. Read that people for example used CC2531 with ZHA without knowing it's not for ready for production. 

Also updated URL for zigpy-cc and zigpy-zigate as both those repositories/libraries have separately moved from independent developers repositories to the collaborative zigpy organization on GitHub.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  Not applicable.
- Link to parent pull request in the Brands repository:  Not applicable.
- This PR fixes or closes issue: Not applicable.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/zha.markdown
Comment (klaasnicolaas) [inline source/_integrations/zha.markdown:44]:
```suggestion
## Compatible hardware

```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by Gamester17** [comment]
Suggestion applied so why is this still blocked now?

**Comment by Gamester17** [comment]
@dmulcahey OK or?

**Comment by Gamester17** [comment]
@klaasnicolaas @dmulcahey @Adminiuga Note that this should preferably go in as soon as possible so many people don't wrongly get a bad initial experience of ZHA just because they tested it with an experimental dongle

**Comment by Adminiuga** [comment]
Agree

**Comment by klaasnicolaas** [comment]
Just wait until we are ready to review and merge your pull request. We do not judge faster if people keep asking for attention ...

**Review by frenck — APPROVED** [review]
Thanks, @Gamester17 👍

**Pull request merged**
